meerk40t/tools/kerftest.py

```diff
@@ -309,7 +309,7 @@ def make_color(idx, maxidx, colidx):
             b = 0
             if maxidx < 8:
                 colrange = 8
-            if maxidx < 16:
```

Comment:
This slightly changes the logic, if our maxidx was below 8 the colrange goes to 8 and not 16.